include/oneapi/tbb/detail/_task.h

```diff
@@ -220,7 +224,11 @@ class alignas(task_alignment) task : public task_traits {
     virtual task* cancel(execution_data&) = 0;
 
 private:
+#if __ARM_ARCH_7A__ || __aarch64__
+    std::uint64_t m_reserved[14]{};
```

Comment:
i decided to put `#if` here, instead of `(task_alignment - sizeof(void*) - sizeof(task_traits)) / sizeof(std::uint64_t)`, because it gets weird on architectures where pointer isn't 8 bytes? idk, this seems cleaner than it was before